omnigibson/robots/tiago.py

```diff
@@ -505,7 +509,9 @@ def camera_control_idx(self):
 
     @property
     def arm_control_idx(self):
-        return {"left": np.array([7, 10, 13, 15, 17, 19, 21]), "right": np.array([8, 11, 14, 16, 18, 20, 22])}
+        return {"left": np.array([7, 10, 13, 15, 17, 19, 21]),
+                "right": np.array([8, 11, 14, 16, 18, 20, 22]),
+                "combined": np.array([7, 8, 10, 11, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22])}
```

Comment:
what is this used for? i think in general the idea was for these idxes to be mutually exclusive to one another. is there a reason why the downstream usecase can't simply concatenate all the idxes from the left and right?

Reply:
After chatting with Minjune and Sujay there seems to be some relevant use cases around this that make it impossible to remove now but we will take care of it as part of #304.